source/_lectures/getting-started-with-sylius.md

```diff
@@ -0,0 +1,50 @@
+---
+name: "Getting started with Sylius"
+
+# číslo uživatele v users.yml
+user: 118
+# @todo: mentors - doplnit?
+
+# obvykle půldenní 3Vladimír Macháček-4 hodiny nebo celodenní 8 (včetně oběda)
+duration: "8 hodin"
+
+# x999, obvykle 3 k za půldenní, 5 k za celodenní
+price: 4999
+
+# max. počet lidí, který zvládneš a je ještě efektinví pro předávání tvého know-how
+# doporučujeme: 6-10
+capacity: "10 lidí"
+
+image: "/assets/images/logos/sylius.png"
+
+perex: "Comprehensive introduction to Sylius E-Commerce framework. Join us and learn how to customize one of the most promising e-commerce solution."
+---
+
+Have you heard about Sylius E-Commerce framework? If so, you know already that it is an important part of Symfony ecosystem and it is one of the biggest, publicly available projects developed in BDD methodology. If not, join me and check that e-commerce development can be a pleasure.
```

Comment:
E-commerce -> e-commerce
you know already -> you already know
join me and check that -> join me and learn the e-commerce technology of the future. Forget legacy platforms like Magento or PrestaShop. Just enjoy being a developer again!